src/test/java/seedu/address/logic/parser/AddressBookParserTest.java

```diff
@@ -119,6 +130,43 @@ public void parseCommand_help() throws Exception {
         assertTrue(parser.parseCommand(HelpCommand.COMMAND_WORD + " 3") instanceof HelpCommand);
     }
 
+    @Test
+    public void parseCommand_groupCreate() throws Exception {
+        GroupCreateCommand command = (GroupCreateCommand) parser.parseCommand(GroupCommand.COMMAND_WORD + " "
```

Comment:
We should create a helper/utility function to help generate these command strings soon haha